internal/configmap/testdata/nfd.kubernetes.io_v1_nodefeaturediscovery.yaml

```
@@ -0,0 +1,123 @@
+apiVersion: nfd.kubernetes.io/v1
+kind: NodeFeatureDiscovery
+metadata:
+  name: nfd-master-server
+  namespace: test-namespace
+spec:
+  instance: "" # instance is empty by default
+  #labelWhiteList: ""
+  #extraLabelNs:
+  #  - "example.com"
+  #resourceLabels:
+  #  - "example.com/resource"
+  operand:
+    image: gcr.io/k8s-staging-nfd/node-feature-discovery:master-minimal
```

minor nit: we can just use the `master` tag, it's the same as `master-minimal`

```suggestion
    image: gcr.io/k8s-staging-nfd/node-feature-discovery:master
```

But maybe we can change this with possible other instances in a separate PR.

---

Done. that field isn't actually being used from that test data, its just using the configData to populate the configmap, so I've changed it to future proof.